lib/iris/tests/results/unit/analysis/cartography/project/TestAll/cube.cml

```diff
@@ -9,24 +9,21 @@
         <dimCoord id="1d45e087" points="[0.0]" shape="(1,)" standard_name="forecast_period" units="Unit('hours')" value_type="float64"/>
       </coord>
       <coord datadims="[2, 3]">
-        <auxCoord id="8f75dad5" points="[[-29.5486755686, -23.2425386265, -27.1960399432,
-		 ..., -26.4183339764, -23.4102889086,
-		 -30.5598035506],
-		[-5.31277829259, -7.60878083145, -22.525325091,
-		 ..., -20.3389138774, -6.63677815264,
-		 -5.96000876128],
-		[11.9373940243, 1.68737606823, -19.7510874124,
-		 ..., -16.5990757719, 3.75281211255,
-		 12.1482277251],
+        <auxCoord id="8f75dad5" points="[[inf, inf, -27.1960378264, ..., -26.4183314681,
+		 inf, inf],
+		[inf, -7.60877872467, -22.5253246282, ...,
+		 -20.3389131851, -6.6367758118, inf],
+		[11.9373960713, 1.68737720645, -19.7510869281,
+		 ..., -16.5990751865, 3.75281337984,
+		 12.1482298878],
 		..., 
-		[84.2615074996, 54.8444519294, 26.7572399859,
-		 ..., 30.1637441878, 58.6051869919,
-		 86.9541960184],
-		[68.173538568, 62.6851805113, 39.6290904051,
-		 ..., 42.324932868, 64.8680482789, 66.5017981217],
-		[43.3491677352, 51.7189413318, 46.2669886567,
-		 ..., 47.2804916021, 51.4709777887,
-		 42.1572752986]]" shape="(10, 10)" standard_name="latitude" units="Unit('degrees')" value_type="float64">
+		[84.2615071705, 54.8444503701, 26.7572382165,
+		 ..., 30.1637424823, 58.6051855118,
+		 86.9541974993],
+		[inf, 62.6851802716, 39.629088106, ...,
+		 42.3249307728, 64.8680485778, inf],
+		[inf, inf, 46.2669891845, ..., 47.2804926757,
+		 inf, inf]]" shape="(10, 10)" standard_name="latitude" units="Unit('degrees')" value_type="float64">
```

Comment:
New `inf` values are an improvement for points outside the Robinson projection - they replace spurious 'nonsense numbers' (https://github.com/OSGeo/PROJ/pull/2151). It is unlikely that this change has negative downstream consequences - @stephenworsley has confirmed the data at these points is already masked and that basic metrics (e.g. mean, stdev) upon regridding are unaffected 👍